src/helpers/useFirebaseFunctions.ts

```diff
@@ -51,7 +54,7 @@ export function useFirebaseFunctions({
       return firebaseFunctions.functionData;
     } catch (err) {
       errorApi.post(err);
-      return [];
+      throw new Error(err);
```

Comment:
If we re-throw this where is it caught?

Reply:
It's an async function injected to the useAsyncRetry hook, so this exception will cause Promise rejection and the error message will be returned in the 'error' property from the hook and displayed on the table instead of the data.

Reply:
Thanks, I wasn't familiar with that hook.